pingouin/parametric.py

```diff
@@ -532,7 +532,8 @@ def rm_anova(data=None, dv=None, within=None, subject=None, correction='auto',
 
     # Calculate sums of squares
     ss_with = ((grp_with.mean() - grandmean)**2 * grp_with.count()).sum()
-    ss_resall = grp_with.apply(lambda x: (x - x.mean())**2).sum()
+    resid = grp_with.apply(lambda x: (x - x.mean()))
```

Comment:
Do you have any documentation for this definition of the residuals? If so, can you add as a comment